### DISCUSSION TITLE
[Esame 18 ottobre 2024] Esercizio 3

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#607] [Esami M-Z]
![3](https://github.com/user-attachments/assets/8716cddd-3a5f-441d-8fa8-0ab25a59817d)

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/0ded8479-f4da-49a3-9fab-61b0a933a054)
![immagine](https://github.com/user-attachments/assets/394df787-94d6-4f33-9066-866244f3f585)

**Comment by niccolo-fato** [comment]
<img width="1280" height="1184" alt="image" src="https://github.com/user-attachments/assets/fb21746e-5fb1-41b7-adb1-816c88191847" />
<img width="1280" height="676" alt="image" src="https://github.com/user-attachments/assets/99388051-3c29-4140-b904-99503735d28a" />
